exports.go

```diff
@@ -7,6 +7,7 @@ import "strconv"
 type ExportList []*papi.IsiExport
 type Export *papi.IsiExport
 
+// Gets a list of all exports on the cluster
 func (c *Client) GetIsiExports() (ExportList, error) {
 	exports, err := c.api.GetIsiExports()
 	if err != nil {
```

Comment:
The closing bracket on this does not appear to be indented to the correct level.